MAUI/DateTimePicker/accessibility.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Accessibility with .NET MAUI Date Time Picker Control | Syncfusion
+description: Learn here all about accessibility with Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) control.
+platform: maui
+control: SfDateTimePicker
+documentation: ug
+---
+
+# Accessibility in .NET MAUI Date Time Picker (SfDateTimePicker)
+
+The .NET MAUI Date Time Picker control has interact with the `Header`, `Column Header`, `Footer`, and `Picker Items`.
+
+## Header Layout
+
+The SfDateTimePicker control has intract header text with localization of header layout. It contains DateFormat and TimeFormat property of the DateTimePickerHeaderView. For example,
```

Comment:
The SfDateTimePicker control has interact header text with localization of header layout. It contains DateFormat and TimeFormat property of the DateTimePickerHeaderView. For example,

Reply:
Changed